tests/util/test_network.py

```diff
@@ -1,42 +1,36 @@
 from __future__ import annotations
 
-import os
-import sys
 from ipaddress import IPv4Address, IPv6Address
 from typing import Type, Union
 
 import pytest
 
-from chia.util.network import IPAddress, get_host_addr
+from chia.util.network import IPAddress, resolve
 
 
 class TestNetwork:
     @pytest.mark.asyncio
-    async def test_get_host_addr4(self):
+    async def test_resolve4(self):
         # Run these tests forcing IPv4 resolution
         prefer_ipv6 = False
-        assert get_host_addr("127.0.0.1", prefer_ipv6=prefer_ipv6) == IPAddress.create("127.0.0.1")
-        assert get_host_addr("10.11.12.13", prefer_ipv6=prefer_ipv6) == IPAddress.create("10.11.12.13")
-        assert get_host_addr("localhost", prefer_ipv6=prefer_ipv6) == IPAddress.create("127.0.0.1")
-        assert get_host_addr("example.net", prefer_ipv6=prefer_ipv6) == IPAddress.create("93.184.216.34")
+        assert await resolve("127.0.0.1", prefer_ipv6=prefer_ipv6) == IPAddress.create("127.0.0.1")
+        assert await resolve("10.11.12.13", prefer_ipv6=prefer_ipv6) == IPAddress.create("10.11.12.13")
+        assert await resolve("localhost", prefer_ipv6=prefer_ipv6) == IPAddress.create("127.0.0.1")
+        assert await resolve("example.net", prefer_ipv6=prefer_ipv6) == IPAddress.create("93.184.216.34")
 
     @pytest.mark.asyncio
-    @pytest.mark.skipif(
-        condition=("GITHUB_ACTIONS" in os.environ) and (sys.platform in {"darwin", "win32"}),
-        reason="macOS and Windows runners in GitHub Actions do not seem to support IPv6",
-    )
-    async def test_get_host_addr6(self):
+    async def test_resolve6(self):
```

Comment:
Doesn't this need the skipif ?

Reply:
Yes of course, thanks! I thought i reverted it. But now i did, really :)